ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -321,6 +321,29 @@ Rectangle {
         }
     }
 
+    function getLineStartPosition(selectionStart) {
+        const text = getPlainText()
+        const lastNewLinePos = text.lastIndexOf("\n\n", messageInputField.selectionStart)
+        return lastNewLinePos === -1 ? 0 : lastNewLinePos + 2
+    }
+
+    function prefixSelectedLine(prefix) {
+        const selectedLinePosition = getLineStartPosition(messageInputField.selectionStart)
+        insertInTextInput(selectedLinePosition, prefix)
+    }
+
+    function unprefixSelectedLine(prefix) {
+        const selectedLinePosition = getLineStartPosition(messageInputField.selectionStart)
+        messageInputField.remove(selectedLinePosition, selectedLinePosition + prefix.length)
+    }
+
+    function isSelectedLineQuoted(selectionStart) {
+        const selectedLinePosition = getLineStartPosition(selectionStart)
+        const text = getPlainText()
+        const lineStartChar = text.charAt(selectedLinePosition)
+        return lineStartChar === ">"
```

Comment:
Shouldn't the line start with "> " instead of only ">"?

Reply:
Yes - nice catch - I changed to "> " in last moment when I was making video and comparing with Design.